9-ComandiAvanzati/res/slides/12b-Soluzione.tex

```diff
@@ -0,0 +1,21 @@
+\begin{frame}[fragile]{Soluzione bibliografia - 1/2}
+
+\begin{block}{Lista}
+    
+\begin{lstlisting}
+\usepackage[natbib = true, backend=biber,style=alphabetic,hyperref,backref]{biblatex}
+\bibliography{bibliografia}
+\begin{document}
+...
+\begin{enumerate}
+\item Prima cit ~\cite{seneca}
+\item Seconda cit ~\cite{nietzsche}
+\item Terza cit ~\cite[p. 42]{abbot}
+\end{enumerate}
+\clearpage
+\printbibliography
+\end{lstlisting}
+
```

Comment:
Toglierei il `\begin{document}` e metterei i puntini anche alla fine dopo `\prinbibliography`. Il primo perchè ormai vorrei che fossero in grado di capire che tutto va racchiuso tra qualcosa, in secondo è perchè non voglio essere troppo speranzoso